### PR TITLE
graalvm8-19.0.0: minor fixes

### PR DESCRIPTION
**Pull request by volth**

### CODE REVIEW

File: pkgs/development/compilers/graalvm/default.nix [hunk 1]
```diff
@@ -363,12 +376,10 @@ in rec {
       cp -rf ${glibc}/lib/* $out/jre/lib/svm/clibraries/linux-amd64/
       cp ${glibc.static}/lib/* $out/jre/lib/svm/clibraries/linux-amd64/
       cp ${zlib.static}/lib/libz.a $out/jre/lib/svm/clibraries/linux-amd64/libz.a
-      # Organize the out dir
-      mkdir -p $out/share && mv $out/man $out/share
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: ```fixupPhase``` does it

File: pkgs/development/compilers/graalvm/default.nix [hunk 2]
```diff
@@ -363,12 +376,10 @@ in rec {
...
     '';
 
-    dontFixup = true; # do not nuke path of ffmpeg etc
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: ```dontFixup``` is a rudiment of working with ```oraclejdk8``` binary distribution
fixupPhase does care on man files and also patches ```#!/bin/sh``` shebangs

File: pkgs/development/compilers/graalvm/default.nix [hunk 3]
```diff
@@ -403,7 +414,7 @@ in rec {
       description = "High-Performance Polyglot VM";
       license = licenses.gpl2;
       maintainers = with maintainers; [ volth hlolli ];
-      platforms = [ "x86_64-linux" ];
```
Comment (volth) [inline pkgs/development/compilers/graalvm/default.nix]: I have not tested yet on "aarch64-linux", but it suppose to work.
Enabled for ofborg to do the aarch64 build too

**Comment by hlolli** [comment]
Just came back from launch, will merge, but wont test locally for now. Let's see what ofborg does!

**Pull request merged**